frontend/src/components/interview/components/history-section/components/interview-note-cards-list/components/interview-note-card/interview-note-card.tsx

```diff
@@ -14,22 +14,22 @@ type Props = {
 const InterviewNoteCard: FC<Props> = ({ note, authorName, postDate }) => {
   return (
     <div className={styles.card}>
+      <p className={styles.noteContent}>{note}</p>
       <div className={styles.cardContentWrapper}>
```

Comment:
```suggestion
      <div className={styles.cardInfoWrapper}>
```
as it does not contain a text now ig it's better to change a class name